back-office/international/translations/translations.md

```diff
@@ -1,12 +1,12 @@
 # Translations
 
-## Conceptual vocabulary
+#### Conceptual vocabulary
 
-- Wordings: original texts found throughout the software. They are in English but may be translated _to_ English for better localization.
-- Translations: wordings translated to a given language.
-- Translation domain: named logic group of wordings. Any given wording is attached to one and only one translation domain. Two identical wordings attached to two different translation domains are considered as two different wordings.
+- **Wordings:** original texts found throughout the software. They are in English but may be translated _to_ English for better localization.
```

Comment:
'They are in English but may be translated to English'... I think there is a problem with this sentence. Maybe you meant 'They are in English but may be translated into other languages for better localization.'?

Reply:
I understand that this is confusing. What I mean is that wordings are actually tokens that bear an approximate relationship with the localized wording. Some projects use complete phrases like "This is a wording", but other use pseudophrases like "this_is_a_wording". Both are simply strings that are used as identifiers. This is why we have English (British and American) translation packages. I have seen examples of wordings that have been rephrased in those packages to sound more natural.

There's probably a better way to explain this :)

Reply:
Ok so if I understand well, you mean that even though the strings are already in English there is still an English translation package so that we can modify the translations for something more natural. Like it was the case for the string 'Please mark each checkbox for the carrier, or carrier, for which you want the payment module(s) to be available.' that became 'Please select available payment modules for every carrier.'

Reply:
Exactly.

Reply:
How about this?
```suggestion
- **Wordings:** original texts found throughout the software. These message tokens are written in English, but they may be translated _to_ English to make them sound more natural in different localizations (eg. British, American, ...).
```

Reply:
```suggestion
- **Wordings:** original texts found throughout the software. Source strings are in English but an English translation package is still available if some wordings need to be rephrased for better localization.
```

Reply:
Sorry I didn't see your suggestion! That's great. Thanks :)